marklogic-data-hub-central/ui/src/components/date-facet/date-facet.tsx

```diff
@@ -70,6 +70,7 @@ const DateFacet: React.FC<Props> = (props) => {
                 onChange={onChange}
                 value={datePickerValue}
                 key={props.name}
+                getCalendarContainer={() => document.getElementById('sideBarContainer') || document.body}
```

Comment:
The getCalendarContainer() property is specific to the date / time picker component. For all other components its not needed and getPopupContainer should be used.